README.md

```diff
@@ -13,33 +13,64 @@
+
+### 3 - HDK Rust
+Holochain applications have been designed to consist at the low-level of WebAssembly (WASM) running in a virtual machine environment. However, most languages will be easiest to use with some stub code to connect into the WASM runtime environment, because of some constraints with WASM. That is the main reason why a "Developer Kit" for a language exists. It is a library, and a syntax, for writing Zome code in that language.
+
+[`hdk-rust`](./hdk-rust) is a solid reference implementation of this, that enables Zomes to be written in the Rust language (the same, somewhat confusingly, as Holochain Core).
+
+Within this repository, some aspects cross over between `core` and `hdk-rust`, such as [core_types](./core_types), since they get stored into WASM memory in `core`, and then loaded from WASM memory, within `hdk-rust`. Related, [wasm_utils](./wasm_utils) is used on both sides to actually perform the storing, and loading, of values into and from WASM memory.
 
+#### Other HDKs and language options
+Any language that compiles to WASM and can serialize/deserialize JSON data can be available as an option for programmers to write Holochain applications.
+
+An HDK for [Assemblyscript](https://github.com/Assemblyscript/assemblyscript) is under experimental development at [`hdk-assemblyscript`](https://github.com/holochain/hdk-assemblyscript). 
+
+We expect many more languages to be added by the community, and there is even an article on how to [write a kit for a new language](https://holochain.github.io/holochain-rust/writing_development_kit.html).
+
+## Documentation: The Book on Holochain
 There is a work-in-progress book of documentation being written about `holochain-rust`. See the published version at the associated GitHub Pages for this repo, [https://holochain.github.io/holochain-rust](https://holochain.github.io/holochain-rust). See instructions for how to contribute to the book at [doc/holochain_101/src/how_to_contribute.md](./doc/holochain_101/src/how_to_contribute.md).
 
 ## Installation & Usage
-The following instructions are for developers of Holochain Core itself. **If you are developing Holochain applications**, you will want to instead proceed to install both the [`hc` command line tool](https://github.com/holochain/holochain-cmd) and the [`hcshell` test runner](https://github.com/holochain/holosqape) to help create Holochain DNA packages suitable for running in a Holochain service.  If you are a Holochain end-user, either you will install DNA packages into a Holochain hApp's service like [HoloSqape](https://github.com/holochain/holosqape), or your application will come with them built in.
 
-**Core Developers Only**
+Please note that this repository, with its installation instructions, is not useful to install if you fall into one of the following two categories:
```

Comment:
Maybe we should start with the "positive" case rather than the "negative"?  i.e. "This repository is for X"   if you are "Y,or Z..." ?